protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/value/processinstance/ProcessInstanceCreationRecord.java

```diff
@@ -109,6 +114,22 @@ public ProcessInstanceCreationRecord setFetchVariables(final List<String> fetchV
     return this;
   }
 
+  public ArrayProperty<StartInstruction> startInstructions() {
+    return startInstructionsProperty;
+  }
+
+  public ProcessInstanceCreationRecord addStartInstructions(
+      final List<StartInstruction> startInstructions) {
+    startInstructions.forEach(this::addStartInstruction);
```

Comment:
❓ Is there a reason why we don't need to copy the `startInstruction` here, but we do in `addStartInstruction`?

Reply:
Because this forwards to the method which copies the entry.